File: MainActivity.java
package com.example.eveytime;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.HorizontalScrollView;
import android.widget.RelativeLayout;
import android.widget.ScrollView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        HorizontalScrollView scrollView1 = findViewById(R.id.main_middle_first_scroll);
        HorizontalScrollView scrollView2 = findViewById(R.id.main_middle_second_scroll);

        scrollView1.setHorizontalScrollBarEnabled(false); // 스크롤 바 없애기
        scrollView2.setHorizontalScrollBarEnabled(false); // 스크롤 바 없애기

        RelativeLayout layout01 = (RelativeLayout) findViewById(R.id.main_middle_second_content1);
        layout01.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Log.d("log01","click layout01");
            }
        });
    }
}

File: MainActivity.kt
package com.android.solution3

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.Fragment
import com.android.solution3.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(R.layout.activity_main)
        setContentView(binding.root)

        val bottomBar = binding.bottomBar

        bottomBar.setOnItemSelectedListener { item ->
            changeFragment(
                when (item.itemId) {
                    R.id.first-> {
                        HomeFragment()
                    }
                    R.id.second-> {
                        SearchFragment()
                    }
                    R.id.third-> {
                        ReelsFragment()
                    }
                    R.id.fourth -> {
                        ShopFragment()
                    }
                    R.id.fifth -> {
                        ProfileFragment()
                    }
                    else -> {
                        HomeFragment()
                    }

                }
            )
            true
        }
        bottomBar.selectedItemId = R.id.first
    }

    private fun changeFragment(fragment: Fragment) {
        supportFragmentManager
            .beginTransaction()
            .replace(R.id.container, fragment)
            .commit()
    }

}